etp-core/etp-backend/.dockerignore

```diff
@@ -0,0 +1,5 @@
+target
+.nrepl-port
+etp-backend.iml
+.cpcache
+.clj-kondo/.cache
```

Comment:
Tämän voisi varmaan tehdä myös niin, että ignoree kaiken ja sen jälkeen kertoo, että mitä ei ignoreta.

Esim:
https://stackoverflow.com/questions/28097064/dockerignore-ignore-everything-except-a-file-and-the-dockerfile

En tiedä olisiko parempi

Reply:
Voisi toimia. Oman kokemuksen mukaan vähemmän käytetty patterni niin saattaa aiheuttaa yllätyksiä kun jotain puuttuukin buildista, mutta olisi varmasti ajan tasalla, vaikka joku esim. avaisi alikansiot idessä omana projektina.

Reply:
Oisko näin kivempi? Lisäsin bonuksena pre-loadin depsuille

Reply:
Tykkään kyllä tuosta enemmän. Tuo pre-load ainakin on hyvä.